definitions/divorce/json/CaseTypeTab/CaseTypeTab-nonprod.json

```diff
@@ -0,0 +1,285 @@
+[
```

Comment:
Copied the whole sub-tab, although I only need a few fields. Again, extra safety.